antora/supplemental-ui/partials/nav-explore.hbs

```diff
@@ -0,0 +1,6 @@
+{{!--
```

Comment:
We also want to set `--nav-panel-menu-height: 100%` in the `:root` of custom.css

Reply:
Good catch, thanks.